common/lib/xmodule/xmodule/course_module.py

```diff
@@ -663,7 +663,14 @@ class CourseFields(object):
         # Ensure that courses imported from XML keep their image
         default="images_course_image.jpg"
     )
-
+    issue_badges = Boolean(
+        display_name=_("Issue OpenBadges"),
```

Comment:
Do we want to explicitly mention "OpenBadges" here or keep it generic in case another badging service is used? 
Also help text in line 669 below.